recipe/build_base.sh

```diff
@@ -269,7 +269,7 @@ if [[ ${_OPTIMIZED} == yes ]]; then
       #         run while on Unix, all 400+ are run, making this slower and less well curated
       _PROFILE_TASK+=(PROFILE_TASK="-m test --pgo")
     else
-      _PROFILE_TASK+=(PROFILE_TASK="-m test --pgo-extended")
```

Comment:
could we add a similar comment to the one of cf regarding why it is ok to run without pgo-extended?

Reply:
Done! Thank you!